tests/sqllogictests/suites/base/09_fuse_engine/09_0001_remote_insert

```diff
@@ -80,3 +80,36 @@ DROP TABLE t2
 
 statement ok
 DROP TABLE t3
+
+## Table Compression:None
+
+CREATE TABLE IF NOT EXISTS t_compression_none(a Boolean, b Timestamp, c Date) Engine = fuse  COMPRESSION = 'NONE';
```

Comment:
Please drop this table before using it.

Reply:
Why do we do this since we drop it when the test tears down? I have not seen the above test drop table first.